tests/Adapter/WorseReflection/Refactor/WorseGenerateAccessorTest.php

```diff
@@ -14,47 +14,70 @@ class WorseGenerateAccessorTest extends WorseTestCase
      */
     public function testGenerateAccessor(
         string $test,
+        string $propertyName,
         string $prefix = '',
         bool $upperCaseFirst = false
     ) {
-        list($source, $expected, $offset) = $this->sourceExpectedAndOffset(__DIR__ . '/fixtures/' . $test);
+        list($source, $expected, $offset) = $this->sourceExpectedAndOffset(
+            __DIR__ . '/fixtures/' . $test
+        );
 
-        $generateAccessor = new WorseGenerateAccessor($this->reflectorFor($source), $this->updater(), $prefix, $upperCaseFirst);
-        $transformed = $generateAccessor->generateAccessor(SourceCode::fromString($source), $offset);
+        $generateAccessor = new WorseGenerateAccessor(
+            $this->reflectorFor($source),
+            $this->updater(),
+            $prefix,
+            $upperCaseFirst
+        );
+        $transformed = $generateAccessor->generate(
+            SourceCode::fromString($source),
+            $propertyName,
+            $offset
+        );
 
         $this->assertEquals(trim($expected), trim($transformed));
     }
 
     public function provideExtractAccessor()
     {
+        $propertyName = 'method';
+
         return [
             'property' => [
                 'generateAccessor1.test',
+                $propertyName,
             ],
             'prefix and ucfirst' => [
                 'generateAccessor2.test',
+                $propertyName,
                 'get',
                 true,
             ],
             'return type' => [
                 'generateAccessor3.test',
+                $propertyName,
             ],
             'namespaced' => [
                 'generateAccessor4.test',
+                $propertyName,
             ],
             'pseudo-type' => [
                 'generateAccessor5.test',
+                $propertyName,
+            ],
+            'multiple-classes' => [
+                'generateAccessor6.test',
+                $propertyName,
```

Comment:
why do we add $propertyName when it is always the same?

Reply:
Would have slightly preferred to keep "constants" as a value in the test rather than repeating here (we don't vary in this test). But not important :)

Reply:
I always try to keep tests configurable, not saying it's a good thing.
It makes my life terrible some time ! :D
But that's why, I just don't like the idea of coupling even... Early optimization, sill got some work to do on that ><